resources/migration-guide-v2-v5.md

```diff
@@ -0,0 +1,444 @@
+# V2 to V5 Migration Guide
+
+`Casper JS SDK V5` introduces significant **breaking changes**, essentially rewriting the SDK from the ground up to enhance usability, align it with Casper SDKs in other languages, and incorporate new features for the Casper 2.0 update.
+
+This guide will walk you through the key changes and provide detailed examples to help you transition your code seamlessly.
+
+---
+
+## Key Highlights of the Migration
+
+1. Simplified APIs and modular design.
+2. Improved alignment with Casper SDKs in other languages.
+3. Enhanced functionality for Casper 2.0.
+4. Transition from deploy-based operations to transaction-based architecture.
+
+---
+
+## Changes and Migration Steps
+
+### 1. **`CasperServiceByJsonRPC` and `CasperClient` Replaced with `RpcClient`**
```

Comment:
`Replaced` or `replaced` with lower case `r`?
Happens in all section titles. Please, check if it's the correct way to write it.